catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/reader/GetRecordsMessageBodyReader.java

```diff
@@ -64,16 +67,26 @@
 
   private DataHolder argumentHolder;
 
+  private QNameMap qNameMap;
+
   public GetRecordsMessageBodyReader(Converter converter, CswSourceConfiguration configuration) {
-    xstream = new XStream(new XppDriver());
+    QName cswQName = new QName(CswConstants.CSW_NAMESPACE_URI, CswConstants.GET_RECORDS_RESPONSE);
```

Comment:
looking at the test records, seems like it all works with test records having both... but how does this handle 
http://www.opengis.net/cat/csw/2.0.2 vs http://www.opengis.net/cat/csw when only 2.0.2 is registered?

Reply:
@millerw8  - there is a unit test that covers the CSW 2.0.1 namespace case - https://github.com/codice/ddf/blob/2.19.x/catalog/spatial/csw/spatial-csw-source-common/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/reader/GetRecordsMessageBodyReaderTest.java#L152.  This one uses the "http://www.opengis.net/cat/csw" namespace.